docs/docs/tutorial/chapter2/getting-dynamic.md

```diff
@@ -69,6 +99,12 @@ Now we'll want to snapshot the schema changes as a migration:
 yarn rw prisma migrate dev
 ```
 
+:::warning
+
+If you are using a MongoDB database you shall not run any migrations as this is not possible with a document database as MongoDB. You want to run `npx prisma generate` instead at this point. To apply the schema to your database you can run `npx prisma db push` which will make collections of your models.
```

Comment:
Just tried to improve the readability a bit here:
```suggestion
If you're using MongoDB, you shouldn't run any migrations since they're not possible with document databases such as MongoDB. You'll want to run `yarn rw prisma generate` at this point instead. To apply the schema to your database, you can run `yarn rw prisma db push`, which will make collections of your models.
```